js/behringer-crave.js

```diff
@@ -0,0 +1,126 @@
+
+    //outputs
+    //row 2
+    {idx: 25, col: 9, row: 1, x: 915, y: 175, name: "LFO TRI", input: false},
+    {idx: 26, col: 10, row: 1, x: 980, y: 175, name: "LFO SQ", input: false},
+    {idx: 27, col: 11, row: 1, x: 1045, y: 175, name: "VC MIX", input: false},
+    {idx: 28, col: 12, row: 1, x: 1110, y: 175, name: "ASSIGN", input: false},
+    {idx: 29, col: 13, row: 1, x: 1177, y: 175, name: "KB", input: false},
+    {idx: 30, col: 14, row: 1, x: 1247, y: 175, name: "GATE", input: false},
+    {idx: 31, col: 15, row: 1, x: 1310, y: 175, name: "VCF", input: false},
+]
+const KNOBS = [
+  {idx: 0, x: 140, y: 293, type: "large", name: "FREQUENCY"},
+  {idx: 1, x: 315, y: 293, type: "large", name: "PULSE WIDTH"},
+  {idx: 2, x: 486, y: 293, type: "large", name: "VCO MOD AMOUNT"},
+  {idx: 3, x: 660, y: 293, type: "large", name: "MIX"},
+  {idx: 4, x: 840, y: 293, type: "large", name: "CUTOFF"},
+  {idx: 5, x: 1012, y: 293, type: "large", name: "RESONANCE"},
+  {idx: 6, x: 1185, y: 293, type: "large", name: "VCF MOD AMOUNT"},
+  {idx: 7, x: 1360, y: 293, type: "large", name: "VOLUME"},
+  {idx: 8, x: 140, y: 487, type: "large", name: "ATTACK"},
+  {idx: 9, x: 315, y: 487, type: "large", name: "DECAY"},
+  {idx: 10, x: 486, y: 487, type: "large", name: "SUSTAIN"},
+  {idx: 11, x: 840, y: 487, type: "large", name: "LFO RATE"},
+  {idx: 12, x: 1185, y: 487, type: "large", name: "GLIDE"},
+  {idx: 13, x: 1360, y: 487, type: "large", name: "VC MIX"},
+  {idx: 14, x: 140, y: 645, type: "large", name: "TEMPO/GATE LENGTH"}
+]
+const KNOB_TYPES = {
+  large: {radius: 30, limit: 310}
+}
+const TOGGLES = [
+  {idx: 0, x: 300, y: 375, type: "twoway", name: "VCO WAVE"},
+  {idx: 1, x: 473, y: 375, type: "twoway", name: "VCO MOD SOURCE"},
+  {idx: 2, x: 645, y: 375, type: "twoway", name: "VCO MOD DEST"},
+  {idx: 3, x: 998, y: 475, type: "twoway", name: "LFO WAVE"},
+  {idx: 4, x: 825, y: 375, type: "twoway", name: "VCF MODE"},
+  {idx: 5, x: 998, y: 375, type: "twoway", name: "VCF MOD SOURCE"},
+  {idx: 6, x: 645, y: 475, type: "twoway", name: "SUSTAIN"},
+  {idx: 7, x: 1344, y: 375, type: "twoway", name: "VCA MODE"},
+  {idx: 8, x: 1980, y: 375, type: "twoway", name: "VCF MOD POLARITY"}
+]
+const TOGGLE_TYPES = {
+  twoway: {values: ["left", "right"]}
+}
+const TOGGLE_WIDTH = 43
+const TOGGLE_HEIGHT = 41
+
+export class BehringerCrave extends FrontPanel {
+  constructor() {
+    super({
+      deviceName: "behringer-crave",
+      canvasWidth: CANVAS_WIDTH,
+      canvasHeight: CANVAS_HEIGHT,
+      connectors: CONNECTORS,
+      connectorOptions: {
+        cellRadius: CELL_RADIUS,
+        cableWidth: CABLE_WIDTH
+      },
+      knobs: KNOBS,
+      knobTypes: KNOB_TYPES,
+      toggles: TOGGLES,
+      toggleOptions: {
+        types: TOGGLE_TYPES,
+        width: TOGGLE_WIDTH,
+        height: TOGGLE_HEIGHT
+      },
+      initialValues: {
+        knobValues: Array(14).fill(150),
```

Comment:
`knobValues: Array(15).fill(158),`

Centers all the knobs on an initial patch and fixes the broken tempo knob.